nimbus-jose-jwt_aws-kms-extension/src/test/java/com/nimbusds/jose/aws/kms/crypto/utils/JWEDecrypterUtilTest.java

```diff
@@ -0,0 +1,129 @@
+package com.nimbusds.jose.aws.kms.crypto.utils;
+
+import com.amazonaws.services.kms.AWSKMS;
+import com.amazonaws.services.kms.AWSKMSClient;
+import com.amazonaws.services.kms.model.*;
+import com.nimbusds.jose.EncryptionMethod;
+import com.nimbusds.jose.JWEAlgorithm;
+import com.nimbusds.jose.JWEHeader;
+import com.nimbusds.jose.RemoteKeySourceException;
+import com.nimbusds.jose.aws.kms.crypto.testUtils.EasyRandomTestUtils;
+import com.nimbusds.jose.aws.kms.exceptions.TemporaryJOSEException;
+import com.nimbusds.jose.jca.JWEJCAContext;
+import com.nimbusds.jose.util.Base64URL;
+
+import java.nio.ByteBuffer;
+import java.util.Map;
+import lombok.var;
+import org.jeasy.random.EasyRandom;
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.junit.platform.commons.support.ReflectionSupport;
+import org.mockito.Mock;
+import org.mockito.MockedStatic;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.Mockito.*;
+import static org.mockito.Mockito.when;
+
+@DisplayName("For the JWEDecrypterUtil class,")
+@ExtendWith(MockitoExtension.class)
+public class JWEDecrypterUtilTest {
+
+    @Nested
+    @DisplayName("the decrypt method,")
+    class DecryptMethod {
```

Comment:
I don't see any success case in the this test-case. Please add them.

Reply:
Added